### PR TITLE
Change Circuit Name by clicking name on TabsBar of page.

### LINKED ISSUE #655: Change Project Name by clicking name on top of page.
**Is your feature request related to a problem? Please describe.**
If I want to change name of project while creating a circuit I have to go to the box at the bottom left, which is visible only in the default setup of page which we can access by deselecting any element, etc.

**Describe the solution you'd like**
I suggest that double clicking the title at the top of the page should make it possible to change the name of the Project. This feature can also be added to the names of circuits.

**Describe alternatives you've considered**
Alternative can be to just add a button beside the title which will enable "Edit Mode" and we'll be able to edit all names like Project, Circuit, etc.

**Are you working on this? (Yes/No)**
Nope. For GCI 2019

### PR DESCRIPTION
**Pull request by Shivansh2407**
Fixes #655 

#### Describe the changes you have made in this pr -
I have fixed the Changing of circuit name by clicking on The circuit name in the above TabsBar. Similar I added the same for Project Name .

### CODE REVIEW

**Comment by Ananthavijay** [comment]
@Shivansh2407 Add a video.

**Comment by Shivansh2407** [comment]
Yes I will be adding it soon @Ananthavijay

**Comment by Shivansh2407** [comment]
I will be squashing the commits soon.

File: public/js/data.js [hunk 1]
```diff
@@ -363,7 +368,10 @@ function generateSaveData(name) {
     name = projectName || name || prompt("Enter Project Name:") || "Untitled";
     data["name"] = stripTags(name)
     projectName = data["name"];
-    setProjectName(projectName)
+    setProjectName(projectName);
+    $( "proj" ).click(function( event ) {
```
[the remark this answers is not in the document]
Comment (sal2701) [reply public/js/data.js]: @Shivansh2407 give this a look

**Comment by Ananthavijay** [comment]
It will be good if there is an icon near the project name. @sal2701 Whatsay?

File: public/js/data.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Shivansh2407) [reply public/js/data.js]: Fixed in my recent commit 👍

**Comment by Shivansh2407** [comment]
Icon of what @Ananthavijay ? Circuit (Minimized) or Circuitverse logo ?

**Comment by Ananthavijay** [comment]
An icon is not needed. But how will the user know that he has to press ````delete```` button instead of ````backspace````

**Comment by Shivansh2407** [comment]
As soon as the textfield is highlighted the User is able to see where to Write so I think we do not require such an indication of Backspace or delete key. Will be glad to hear your views too.

**Comment by satu0king** [comment]
Please resolve the conflicts, this is a much needed feature

**Comment by Shivansh2407** [comment]
Sure @satu0king 👍

**Comment by aayush-05** [comment]
@Shivansh2407 are you working on this ?

**Comment by satu0king** [comment]
@Shivansh2407 can you redirect this PR to new_UI branch? Thanks!

**Pull request closed without merging**